variables.tf

```diff
@@ -318,3 +318,15 @@ variable "enable_host_encryption" {
   type        = bool
   default     = false
 }
+
+variable "node_resource_group" {
+  description = "The auto-generated Resource Group which contains the resources for this Managed Kubernetes Cluster."
```

Comment:
Could we add "Azure requires that a new, non-existent Resource Group is used, as otherwise the provisioning of the Kubernetes Service will fail." to `description` too?

[doc](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#node_resource_group)

Reply:
Could we add "Azure requires that a new, non-existent Resource Group is used, as otherwise the provisioning of the Kubernetes Service will fail." to `description` too?

[doc](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#node_resource_group)